Gemfile.lock

```diff
@@ -6,14 +6,14 @@ GEM
     nokogiri (1.6.8.1)
       mini_portile2 (~> 2.1.0)
     puma (3.6.0)
-    rack (1.6.5)
+    rack (1.5.2)
```

Comment:
I was having some problems disabling the rack protection "json_csrf" which we already have working in the branch "disable-protection", so in the debug process I was locking up the gem versions with the same versions that we have in that old branch. 

Finally I got it working, and I thought the problem was a docker cache problem. But now seeing the rack downgrade, maybe the problem was the rack version. Let me review that so I can be sure.